include/aikido/planner/vectorfield/MetaSkeletonStateSpaceVectorField.hpp

```diff
@@ -0,0 +1,68 @@
+#ifndef AIKIDO_PLANNER_VECTORFIELD_METASKELETONSTATESPACEVECTORFIELD_HPP_
+#define AIKIDO_PLANNER_VECTORFIELD_METASKELETONSTATESPACEVECTORFIELD_HPP_
+
+#include <aikido/planner/vectorfield/VectorField.hpp>
+#include <aikido/planner/vectorfield/VectorFieldPlannerStatus.hpp>
+#include <aikido/statespace/dart/MetaSkeletonStateSpace.hpp>
+
+namespace aikido {
+namespace planner {
+namespace vectorfield {
+
+/// This class defines a vector field in a configuration space.
+///
+/// Any vector field should inherit this class to implememnt functions
+/// for calculating joint velocities giving joint positions and time,
+/// and checking planning status.
+class MetaSkeletonStateSpaceVectorField : public VectorField
```

Comment:
I don't think this class is necessary - it doesn't seem to provide any value over `VectorField` itself. I think we can safely roll all of this functionality into `BodyNodePoseVectorField`.